recipes-app/node-red-node-random/node-red-node-random-0.4.1.bb

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) Siemens AG, 2023
+#
+# Authors:
+#  Li Hua Qian <huaqian.li@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+inherit npm
+inherit node-red-module
+
+DESCRIPTION = "A set of random nodes for Node-RED"
```

Comment:
Commit message lacks content - including the reason why we want this node.

Reply:
It's per customer demand -- I write in the commit message what this node could do, hope that helps.